src/lib/tuya.ts

```diff
@@ -440,6 +440,13 @@ export const valueConverter = {
     trueFalseEnum0: valueConverterBasic.trueFalse(new Enum(0)),
     trueFalseEnum1: valueConverterBasic.trueFalse(new Enum(1)),
     onOff: valueConverterBasic.lookup({'ON': true, 'OFF': false}),
+    onOffWithDelay: {
+        to: async (value: string) => {
+            await utils.sleep(200);
```

Comment:
I put delay here, but that caused that all requests are delayed. It should look like `utils.sleep(transitiont * 200)` but I don't know how proper declare `transition`

Reply:
What do you mean with `transitiont`?

Reply:
When I will change switch in Z2M for more then one entity at one time, I want to set some number like 0, 1, 2, 3 for each. I will get delay time such 0 * 200 ms, 1 * 200 ms, 2 * 200 ms and so on.

Reply:
If you change `to: async (value: string) => {` to `to: async (value: string, meta) => {`, you can get the message with `meta.message`, from that you can count the number of `state_` and create the timeout based on that.